graphql/errors.go

```diff
@@ -25,6 +26,10 @@ func (e ClientError) SanitizedError() string {
 	return e.message
 }
 
+func (e ClientError) Unwrap() error {
+	return e.inner
+}
```

Comment:
maybe just `SafeError` needs it? `ClientError` is only ever emitted by thunder and never wraps

Reply:
agreed, updating